src/main/java/TodoList.java

```diff
@@ -0,0 +1,88 @@
+public class TodoList {
+    static final String LINE_BREAK = "____________________\n";
+    static final String YOUR_TODO_LIST = "This is your todo list.\n";
+    static final String HAVE_ADDED = "I have added ";
+    static final String TO_LIST = " to your todo list.\n";
+    static final String MARKED_TASK = "I have marked Task ";
+    static final String UNMARKED_TASK = "I have unmarked Task ";
+    static final String IN_LIST = " in your todo list.\n";
+    static final int MAX_COMMAND_ARGS = 3;
+    static final int MAX_TASKS = 100;
+    static final String BY_COMMAND = " /by";
+    static final int BY_COMMAND_LENGTH = 5;
+    static final String FROM_COMMAND = " /from";
+    static final int FROM_COMMAND_LENGTH = 7;
+    static final String TO_COMMAND = " /to";
+    static final int TO_COMMAND_LENGTH = 5;
+    private static int tasksNumber = 0;
```

Comment:
This looks like it indicates the number of tasks, but may be misinterpreted to be the index of a specific task. Can consider renaming to numberOfTasks or something similar.